CHANGES/6383.feature

```diff
@@ -0,0 +1 @@
+Added ``network_interface`` parameter to ``TCPConnector`` to allow binding directly to an interface.
```

Comment:
In our project, you are also welcome to have a byline in the changelog, like this:
```suggestion
Added the ``network_interface`` parameter to :class:`~aiohttp.TCPConnector` to allow binding it directly to a source interface — by :user:`Mostafa-Mahdi`.
```
Also, let's try to link `TCPConnector` from the changelog to make it clickable. You can preview this after making the change via https://aiohttp--7132.org.readthedocs.build/en/7132/changes.html#features.